### ISSUE TITLE
Issues with PostgreSQL dump files

### ISSUE THREAD

**Issue by rjcf18** [joomla/joomla-cms#17864]
### Steps to reproduce the issue

Download CMS from repo and attempt the manual installation. Place the correct DB info on the configuration.php file in the base folder like this for example:

```	
public $dbtype = 'PostgreSQL';
public $host = '127.0.0.1';           
public $user = 'postgres';            
public $password = 'password';  
public $db = 'joomla';
public $dbprefix = 'jos_';
```

and after importing joomla.sql file in the installation/sql/postgresql folder into postgres try importing the sample_testing.sql dump file to see the sql errors while importing and check the joomla instance in the browser to see the errors. The first issues are with escaping quotes (') on strings, another issue with date fields during the import process and after solving those issues and have the dump files imported without errors throughout the import there are still errors in the instance.

### Expected result

Inserting fine and finish the installation successfully having a Joomla instance with no errors.

### Actual result

Installation is not done properly and several errors appear:

![deepinscreenshot_select-area_20170904135831](https://user-images.githubusercontent.com/6710380/30027576-751c933c-9179-11e7-8361-eced022fdaab.png)

![deepinscreenshot_select-area_20170902215840](https://user-images.githubusercontent.com/6710380/30027589-7ec04d20-9179-11e7-9138-712d04fcbbc2.png)

![deepinscreenshot_select-area_20170902215815](https://user-images.githubusercontent.com/6710380/30027593-81176586-9179-11e7-9f89-49fddd8a1533.png)

![deepinscreenshot_select-area_20170904150014](https://user-images.githubusercontent.com/6710380/30029628-e72d9590-9181-11e7-9886-9634ef5cd08b.png)

### System information (as much as possible)

Joomla! 4 and Joomla! 3.X
OS: CentOS Linux release 7.3.1611 (Core)
Kernel: 4.4.0-53-generic
Google Chrome: 60.0.3112.78 (64 bit)
PHP: 7.1.8
PostgreSQL: 9.6.4

**Comment by ghost** [comment]
closed as having Pull Request #17869 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17864">issues.joomla.org/tracker/joomla-cms/17864</a>.</sub>

**Comment by alikon** [comment]
please reopen cause #17869 don't fix all is a "little step"

**Comment by ghost** [comment]
reopened as stated above.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17864">issues.joomla.org/tracker/joomla-cms/17864</a>.</sub>

**Comment by ghost** [comment]
closed as having Pull Request #17891<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17864">issues.joomla.org/tracker/joomla-cms/17864</a>.</sub>

**Comment by alikon** [comment]
sorry @franz-wohlkoenig  
again, please don't close is not enough
just another little step  :smile:

**Comment by ghost** [comment]
closed as having Pull Request #17897<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17864">issues.joomla.org/tracker/joomla-cms/17864</a>.</sub>
